addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -44,7 +44,7 @@
                     </div>
                     <div class="col-lg-8 o_mail_header_social" style="text-align: right;" t-translation="off">
                         <a title="Facebook" href="https://www.facebook.com/Odoo">
-                            <span class="fa fa-facebook rounded-circle text-white" style="background-color: rgb(43, 59, 88) !important;"/>
+                            <span class="fa fa-facebook ​rounded-circle text-white" style="background-color: rgb(43, 59, 88) !important;"/>
```

Comment:
I don't see anything different on this line. What am I missing ?

Reply:
It was a non-breakable space or something of the sort. It's been removed and thrown to the :fire: